packages/lib/src/components/internal/DisclaimerMessage/DisclaimerMessage.scss

```diff
@@ -1,12 +1,13 @@
 @import "../../../style/index";
 
 .adyen-checkout-disclaimer__label {
-    position: relative;
+    width: 100%;
+    text-align: center;
```

Comment:
This cause the T&Cs to align centrally. 
For the Card component they should be left aligned. 
I don't know about for OnlineBanking_PL or Donation component?

Reply:
For OnlineBanking_PL or Donation component, the design is to align it centrally. I'll remove the `text-align: center;` from this component, and let the parent decides how to position it.